src/org/thoughtcrime/securesms/components/ThumbnailView.java

```diff
@@ -268,7 +271,7 @@ protected Boolean doInBackground(Void... params) {
           protected void onPostExecute(Boolean success) {
             GlideRequest request = applySizing(glideRequests.load(new DecryptableUri(thumbnailUri))
                 .diskCacheStrategy(DiskCacheStrategy.NONE)
-                .transition(withCrossFade()), new CenterCrop());
+                .transition(withCrossFade()), fit);
```

Comment:
We get a merge conflict here because of https://github.com/deltachat/deltachat-android/pull/1824.

Also, this change is probably supposed to prevent stickers from being cropped; at least in my tests, the stickers returned by `adb_bot1@testrun.org` _were_ cropped (of course the reason could also be that the bot cropped the pictures).

I don't know how to get rid of cropping for just stickers; generally, cropping is defined in `thumbnail_view.xml` (`android:scaleType="centerCrop"`)

BTW it is also possible that I deleted then-unused code in https://github.com/deltachat/deltachat-android/pull/1824 that is now not unused anymore, but you will see :)